libs/domains/clusters/feature/src/lib/cluster-delete-modal/cluster-delete-modal.tsx

```diff
@@ -71,148 +90,167 @@ export function ClusterDeleteModal({ cluster }: ClusterDeleteModalProps) {
 
   const ctaButtonDisabled =
     (kubernetes === 'SELF_MANAGED' ? confirmKubernetesChecked !== true : confirmCloudProviderChecked !== true) ||
-    confirmQoveryConfigChecked !== true
+    confirmQoveryConfigChecked !== true ||
+    // WARN: Cannot use react-hook-form isValid due to
+    // https://github.com/react-hook-form/react-hook-form/issues/2755
+    watch('name') !== 'delete'
 
   return (
-    <ModalConfirmation
-      title="Confirm deletion cluster"
-      name={name}
-      callback={async () => {
-        try {
-          await deleteCluster({
-            organizationId,
-            clusterId,
-            clusterDeleteMode: cluster.kubernetes === 'SELF_MANAGED' ? 'DELETE_QOVERY_CONFIG' : clusterDeleteMode,
-          })
-          navigate(CLUSTERS_URL(organizationId))
-        } catch (error) {
-          console.error(error)
-        }
-      }}
-      ctaButtonDisabled={ctaButtonDisabled}
-      isDelete
-    >
-      <div className="mb-6 rounded border border-red-500 bg-red-50 p-4 text-sm text-neutral-400">
-        {cluster.kubernetes !== 'SELF_MANAGED' && (
-          <InputSelect
-            className="mb-3"
-            label="Delete mode"
-            options={clusterDeleteModeOptionsFormatted}
-            onChange={(value) => setClusterDeleteMode(value as ClusterDeleteMode)}
-            value={clusterDeleteMode}
-          />
-        )}
-        <p className="font-medium">
-          Use it carefully this action is irreversible. Please confirm that you are aware of the operation impact:
-        </p>
-
-        <p className="flex flex-col">
-          <div className="flex flex-row gap-3 py-2">
-            <Checkbox
-              name="confirm_qovery_config"
-              id="confirm_qovery_config"
-              className="shrink-0"
-              color="red"
-              checked={confirmQoveryConfigChecked}
-              onCheckedChange={setConfirmQoveryConfigChecked}
+    <div className="p-6">
```

Comment:
Could you use `Section` and `Heading` component?

Reply:
I cannot use `Section` and `Heading` because it generates an `h1`
![2024-08-23-150331_1414x892_scrot](https://github.com/user-attachments/assets/07ef0f47-b90c-4d7d-a7e1-e01e98374ead)
and this is not valid semantically
https://stackoverflow.com/a/38467898

If we want to allow that, we need to add a `Section` wrapper to https://github.com/Qovery/console/blob/a541e41cddcb1ef7b735eff28c6016d69a31ca1d/libs/shared/ui/src/lib/components/modal/modal-root.tsx#L91
but I don't want to do it because it might affect other modals